### PR TITLE
#CX-10203 - fix(accessibilty): updated aria-attributes in modal

### PR DESCRIPTION
**Pull request by iruchipriya**
CX-10203 

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Vidcast - https://app.vidcast.io/share/5c705a08-46e8-45d6-a423-d84de553a3b3

### CODE REVIEW

File: web-components/src/components/modal/Modal.ts [hunk 1]
```diff
@@ -272,8 +272,8 @@ export namespace Modal {
                   id="${this.htmlId}"
                   class="md-modal ${classMap(this.modalContainerClassMap)}"
                   aria-label="${this.ariaLabel}"
-                  aria-labelledby="modal_header"
-                  aria-describedby=${ifDefined(this.ariaDescription)}
+                  aria-labelledby=${ifDefined(this.ariaLabelledBy || undefined)}
```
Comment (neelabhnagaich) [inline web-components/src/components/modal/Modal.ts:275]: why are we removing "modal_header" from aria-lablledby as heading needs to be linked to it or can we keep it as default if this needs to be made dynamic ?

Comment (iruchipriya) [reply web-components/src/components/modal/Modal.ts:275]: Hi, this was introduced by me as a fix for this story only. We do not need this now. So keeping it dynamic only as this.ariaLabelledBy.
Prev PR - https://github.com/momentum-design/momentum-ui/pull/1555/files

Comment (neelabhnagaich) [reply web-components/src/components/modal/Modal.ts:275]: The problem I see with this change is it resulted in fixing multiple modal not just station credential modal, you might need to verify other modals if their behavior is not impacted with this change.

Comment (iruchipriya) [reply web-components/src/components/modal/Modal.ts]: updated the initial value. It won't impact other modal behaviour now.

**Review by vinitkumargoel — APPROVED** [review]

**Review by neelabhnagaich — APPROVED** [review]

**Pull request merged**